### PR TITLE
Sampled Values

### PR DESCRIPTION
**Pull request by TomSeymour**
Note:  this includes the GetSampleFiltered methods which are coming soon.

### CODE REVIEW

File: Documentation/SequentialDataStore/Reading_Data.md [hunk 1]
```diff
@@ -146,6 +147,7 @@ Data transformations are supported for all single stream reads, but transformati
     api/v1-preview/Tenants/{tenantId}/Namespaces/{namespaceId}/Streams/{streamId}/Data/Transform
     api/v1-preview/Tenants/{tenantId}/Namespaces/{namespaceId}/Streams/{streamId}/Data/Transform/Interpolated
     api/v1-preview/Tenants/{tenantId}/Namespaces/{namespaceId}/Streams/{streamId}/Data/Transform/Summaries
+    api/v1-preview/Tenants/{tenantId}/Namespaces/{namespaceId}/Streams/{streamId}/Data/Transform/Sampled
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/Reading_Data.md]: Do we support this? 
Would we apply the stream view, then the sample the transformed data? What happens if the sampleBy property is not in the target type?

**Comment by mckenzielee** [comment]
@osimarkrogers We need to wait on the merge of this change.

File: Documentation/SequentialDataStore/Reading_Data_API.md [hunk 2]
```diff
@@ -1267,6 +1267,126 @@ Content-Type: application/json
 ```
 ****
 
+## ``Get Samples``
+
+Returns data sampled by intervals between a specified start and end index. 
+  
+Sampling is driven by a specified Property or Properties of the stream's Sds Type.  Property types that cannot be interpolated do not support sampling requests. Strings are an example of indexes that cannot be interpolated.
```
[the remark this answers is not in the document]
Comment (TomSeymour) [reply Documentation/SequentialDataStore/Reading_Data_API.md]: I believe the need to interpolate comes from the SdsBoundaryType (ExactOrCalculated)  which is a valid setting.  Additionally we are searching for maxs and mins, and in doing so we get the Double equivalent of the values before comparing them.  I am not sure what would happen with properties that couldn't be represented in a numerical form.

File: Documentation/SequentialDataStore/Reading_Data.md (hunk 1 above)
Comment (gdinak) [reply Documentation/SequentialDataStore/Reading_Data.md]: Yes.
Yes.
BadRequest.

**Pull request merged**